next_generation/README.rst

```diff
@@ -32,9 +32,44 @@ Follow the following steps to run these unittests:
 
      source test_venv/bin/activate
 
-3. Run the tests
+3. Run the unittests
 
    .. code-block::
 
-     python -m pytest
+     python -m pytest pint_server/tests/unit
 
+
+Running the Functional Tests
+------------------------------
+Follow the following steps to run the functional tests:
+
+Pre-requisite
+These functional tests expect the environment under test to be setup correctly.
+
+1. Setup a python virtual environment
```

Comment:
this currently has to be run from the next_generation folder, which presumably will be the top level folder eventually... though I suppose since this readme is in that folder, I probably should have assumed as much

Reply:
Yes, we'll eventually move it to the top level as the Sinatra code will be removed in favor of the Flask app.